llm-agents/src/steps/url_scraper.py

```diff
@@ -36,9 +36,15 @@ def url_scraper(
     Returns:
         List of URLs to scrape.
     """
-    examples_readme_urls = get_nested_readme_urls(repo_url)
-    docs_urls = get_all_pages(docs_url)
-    website_urls = get_all_pages(website_url)
-    return (
-        docs_urls + website_urls + examples_readme_urls + [release_notes_url]
+    # examples_readme_urls = get_nested_readme_urls(repo_url)
+    # docs_urls = get_all_pages(docs_url)
+    # website_urls = get_all_pages(website_url)
+    # all_urls = docs_urls + website_urls + [release_notes_url]
```

Comment:
Remove

Reply:
I wanted to keep this as an option for people to include a greater number of documents as data sources for the vector store.